src/common/templates/_databaseconnectionsv2.tpl

```diff
@@ -772,3 +772,36 @@ Define environment variable value based on ENABLE_ELASTIC
   {{- printf "%s" (split ":" (index $hosts 0))._0 }}
   {{- end }}
 {{- end }}
+
+
+{{- define "harnesscommon.dbconnectionv3.postgresPort" }}
```

Comment:
added this change as it might be needed by data pl team near time soon and was missing.